app/services/taxonomy/show_page.rb

```diff
@@ -67,5 +67,20 @@ def associated_taxons
     def taxon_deletable?
       taxon.content_id != GovukTaxonomy::ROOT_CONTENT_ID
     end
+
+    def email_subscribers
+      @email_subscribers ||= begin
+        email_lists = []
+
+        begin
+          email_lists = Services.email_alert_api.find_subscriber_list(links: { taxon_tree: [taxon.content_id] })
+        rescue GdsApi::BaseError, SocketError => e
+          GovukError.notify(e)
+        end
+
+        subscriptions = email_lists.dig(0, "active_subscriptions_count")
+        subscriptions.present? ? subscriptions : "?"
```

Comment:
Is there a reason why you're using "?" instead of a more descriptive "no subscriptions found"-type message?

Reply:
The "?" only shows up if content-tagger can't talk to email-alert-api (if it can and there are no subscribers, you'd get "0" rather than "?").

Reply:
Probably the simplest way is to let email-alert-api return a 500 error:

```
stub_request(:get, "https://email-alert-api.test.gov.uk/subscriber-lists")
   .to_return(body: [].to_json, status: 500)
```

This should cause gds-api-adapters to raise a `GdsApi::ServerError` (or something).